Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -68,6 +70,10 @@ extension ConfigureStrings: CustomStringConvertible {
                 "trying to get the default instance. More info here: https://errors.rev.cat/configuring-sdk"
         case .sdk_version(let sdkVersion):
             return "SDK Version - \(sdkVersion)"
+        case .invalidAPIKey:
+            return "The specified API Key is not recognized.\n" +
+            "Ensure that you are using the public app-specific API key.\n" +
+            "See https://docs.revenuecat.com/docs/authentication for more details."
```

Comment:
maybe we could hint at the possibility of having copy/pasted the wrong key? 
or even say something like "the api key should start with `_appl`" or "the api key should look like `_appl1a2b3c4d5e6f7h`"